android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -255,6 +255,10 @@ protected void onReadyToClose() {
         finish();
     }
 
+    protected void onTranscriptionChunkReceived(HashMap<String, Object> extraData) {
+        JitsiMeetLogger.i("Transcription chunk received: " + extraData);
```

Comment:
Don't put this here. Put it in the main activity, and commented out.

Reply:
I see that all the other events like CONFERENCE_TERMINATED or READY_TO_CLOSE are here and MainActivity extends JitsiMeetActivity.

Reply:
READY_TO_CLOSE is special because it terminates the activity. The rest shouldn't really be there...

Reply:
Can we leave it as it is right now and do a separate PR where we tackle all the events after this?